data/json/enchantments.json

```diff
@@ -11,5 +11,17 @@
     "condition": "ACTIVE",
     "ench_effects": [ { "effect": "invisibility", "intensity": 1 } ],
     "emitter": "emit_shadow_field"
+  },
+  {
+    "type": "enchantment",
+    "id": "ENCH_BIO_FLASHLIGHT",
+    "condition": "ACTIVE",
+    "values": [ { "value": "LUMINATION", "add": 60 } ]
```

Comment:
Is this already documented somewhere and if not could you add it with this PR? Like what "add" and the number mean?

Reply:
Its documented in MAGIC.md under enchantments